notebooks/116-sparsity-optimization/116-sparsity-optimization.ipynb

```diff
@@ -8,7 +8,9 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.      -load_config bert_80pc_sparse_quantized_ir/perf_config.json)
</pre><p>probably commandline should reuse directory for avoid confusion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
@eaidova can you elaborate on this? I checked and confirmed that we only use this folder in the notebook. Are you actually referring to use variable here? As far as I am aware, this is not possible because this is bash command, it cannot access python variable.

Reply:
it is possible to use python variable in bash commands in notebook with $ operator, please look on other notebooks

Reply:
Changed in the latest commit https://github.com/openvinotoolkit/openvino_notebooks/pull/903/commits/3ccdb2ed174c113ba1dbcb3aa176510342f08a95 . Thanks for pointing to a reference @eaidova . It helped and I learnt something new!